src/cmd/flux-start.c

```diff
@@ -85,27 +96,14 @@ static int no_caliper_fatal_err (optparse_t *p, struct optparse_option *o,
 
 const char *usage_msg = "[OPTIONS] command ...";
 static struct optparse_option opts[] = {
-    { .name = "verbose",    .key = 'v', .has_arg = 0,
-      .usage = "Be annoyingly informative", },
+    { .name = "verbose",    .key = 'v', .has_arg = 2, .arginfo = "[LEVEL]",
```

Comment:
> Why not take multiple -v supplied on the command line to increase verbosity as we do with other commands?

`-v -v` is handled as equivalent to `--verbose=2` here.  The argument is already optional.

> One drawback of having an option that takes an optional argument is that it can't be combined with other short options

Yeah, that is a bit unfortunate, so `-vv` will fail as a bad argument to `-v`.

Reply:
> -v -v is handled as equivalent to --verbose=2 here. The argument is already optional.

Yeah, my question is why the difference here compared to all the other flux commands that take multiple `-v` where `-vv` works? Did you need `--verbose=2` explicitly for some reason? It broke for me immediately since I tried `flux start -vvs 4`

Reply:
Ah, is `-vv` support preferable to `--verbose=2` support?  I leaned the other way for no apparent reason.  Maybe we could have it both ways by registering the short and long options separately?

Reply:
Yeah, sorry I thought you were aware of that. It is how we use it in `flux-shell` `flux-exec` and `flux mini run`.

Good thought about registering long and short options separately, unfortunately I'm not sure we ever added support for short-option-only options in liboptparse.

You could, however, have a separate option `--loglevel` or something if there is a pressing need to set the verbosity numerically?

Reply:
I'll go ahead and get rid of the optional argument.  I think you're right that `-vv` is the more common and expected idiom.

Reply:
perhaps eventually liboptparse could support a flag to make the optional argument apply only to the long option, perhaps using your idea of separating the short from the longopts in the underlying `getopt_long()` configuration.

One other drawback to optional arguments on long opts, however, is that unlike required arguments, the `=` is compulsory if you want to supply an argument, i.e. `--verbose 2` wouldn't work. Perhaps there would be a way to work around this limitation as well.